oojs.js

```diff
@@ -10,3 +10,23 @@
   // Have an artist property that is determined by some input.
 
 // Create an instance of the Album class.
+
+class Playlist{
+  constructor(title, song){
+    this.title = title;
+    this.song = song;
+    }
+  addSong(){
+  return `Your song was added to the Playlist ${this.song}`;
+  }
+}
+const music = new Plalist("Music");
+music.addSong();
+
+class Album{
+  constructor(artist, song){
+    super(artist, song);
+    this.artist = artist;
+    this.song = song;
```

Comment:
Creating the songs array is handled in the `Playlist` constructor so this line should be deleted.